src/chainparams.cpp

```diff
@@ -133,7 +133,7 @@ class CMainParams : public CChainParams {
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight = 1046400;
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].hashActivationBlock =
             uint256S("00000000002038016f976744c369dce7419fca30e7171dfac703af5e5f7ad1d4");
-        consensus.vUpgrades[Consensus::UPGRADE_NU5].nProtocolVersion = 170017;
+        consensus.vUpgrades[Consensus::UPGRADE_NU5].nProtocolVersion = 170100;
```

Comment:
ZIP 252 needs to be updated to reflect these changes to `chainparams.cpp`.

Reply:
This is now https://github.com/zcash/zips/pull/600

Reply:
The ZIP update PR was merged.